ci-operator/step-registry/ipi/conf/aws/proxy/ipi-conf-aws-proxy-commands.sh

```diff
@@ -246,7 +230,9 @@ Outputs:
 EOF
 }
 
-export AWS_SHARED_CREDENTIALS_FILE="${CLUSTER_PROFILE_DIR}/.awscred"
+if test -f "${CLUSTER_PROFILE_DIR}/.awscred"; then
+  export AWS_SHARED_CREDENTIALS_FILE="${CLUSTER_PROFILE_DIR}/.awscred"
```

Comment:
Instead of only exporting this when the file exists, I'd rather make the lack of this file fatal with something like:

```sh
export AWS_SHARED_CREDENTIALS_FILE="${CLUSTER_PROFILE_DIR}/.awscred"
if ! test -f "${AWS_SHARED_CREDENTIALS_FILE}"
then
  echo 'No AWS credentials provided' >&2
  exit 1
fi
```

That way we die with a clear message before getting down to the `aws` calls and failing there.

Reply:
It'd be nice as a general rule if our scripts/tools were runnable outside of Prow easily.  If for example I have my aws config at `~/.aws/credentials` - that's the standard place and I should be required to set an environment file to use it.

Reply:
> It'd be nice as a general rule if our scripts/tools were runnable outside of Prow easily.

Can you run this one outside of Prow easily with this?  You'd have to set up an install-config with explicit subnets, a cluster profile directory with `ssh-publickey`, etc.  I'd rather prioritize "fails with a clear message when we bork the Prow invocation or break cross-step APIs".